model-desc/ins-model-props.yml

```diff
@@ -416,4 +417,50 @@ PropDefinitions:
     Desc: any terms related to the dataset study
     Type: string  # list
     Req: false  # may not be available
-    Private: false
\ No newline at end of file
+    Private: false    
+  # Properties of file:
+  file_id:
+    Desc: Unique file identifier (UUID)
+    Type: string
+    Key: true
+    Req: true
+    Private: false
+  file_name:
+    Desc: The literal label for an electronic data file.
+    Term:
+      - Origin: caDSR
+        Code: "11284037"
```

Comment:
Agreed on your point about CDEs. If we have them for new properties, it makes sense to add them. I don't see the harm, though I don't think the effort is worth it to backfill or update old properties with CDEs right now.